test/e2e-common.sh

```diff
@@ -31,14 +31,15 @@ readonly ISTIO_VERSION="stable"
 readonly ISTIO_MESH=0
 # Namespaces used for tests
 export TEST_NAMESPACE="${TEST_NAMESPACE:-knative-operator-testing}"
-export SYSTEM_NAMESPACE=${TEST_NAMESPACE}           # knative-serving
-export TEST_EVENTING_NAMESPACE=${TEST_NAMESPACE}    # knative-eventing
-export TEST_RESOURCE="knative"    # knative-eventing
+export SYSTEM_NAMESPACE=${TEST_NAMESPACE}
+# TODO: remove when components can coexist in same namespace
```

Comment:
Is there an immediate plan for this? I don't know if we need a TODO in place unless it's planned soon.

Reply:
I do not think so. serving and eventing have to be under different namespaces.